rules/framework.bzl

```diff
@@ -159,22 +160,20 @@ def _get_virtual_framework_info(ctx, framework_files, compilation_context_fields
     # We need to map all the deps here - for both swift headers and others
     fw_dep_vfsoverlays = []
     for dep in transitive_deps + deps:
+        # Collect transitive headers. For now, this needs to include all of the
+        # transitive headers
+        if not CcInfo in dep:
+            continue
+        for h in dep[CcInfo].compilation_context.headers.to_list():
+            propagated_interface_headers.append(depset([h]))
```

Comment:
will h.is_source be true for headers? Or true only to a set of headers? The original logic of continue if is_source is baffling me...